### PR TITLE
feat(style): modify advance panel ui

### PR DESCRIPTION
**Pull request by wenmine**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

### What is the new behavior?
1. 修改了按钮之间的间距问题，以及各个弹窗的底部按钮之间的间距问题
2. 根据UI调整了右侧高级面板：1. 按钮的位置 2. 提示 3. 高级设置的间距、编辑代码的按钮 4. 新增事件弹窗样式调整 5. 已有事件的popover样式调整

image
image
image
image
image

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Release Notes

- **New Features**
  - Enhanced button layouts and spacing across various components for improved user experience.
  - Introduced new classes for better margin and layout control in the UI.
  - Added draggable functionality and informational tips in dialog components.

- **Bug Fixes**
  - Updated dialog titles for clarity and improved user understanding.

- **Styling Changes**
  - Adjusted CSS properties for better visual consistency and spacing in components.
  - Updated color schemes for event-related UI elements to enhance readability.

- **Documentation**
  - Added informational tips in dialog components to guide user interactions.

### CODE REVIEW

**Comment by chilingling** [comment]
@coderabbitai review

File: packages/plugins/i18n/src/Main.vue [hunk 1]
```diff
@@ -485,8 +485,11 @@ export default {
     margin-right: 6px;
     color: var(--ti-lowcode-i18n-icon-color);
   }
+  .middle-btn {
+    margin-left: 0;
```
Comment (chilingling) [inline packages/plugins/i18n/src/Main.vue:489]: 这应该是个共性问题。
在公共主题样式，我们添加了默认按钮右边 8px 的距离。

```less
.tiny-button {
  &.tiny-button-default {
   // 这里设置了右边距 8px
    margin-right: 8px;
  }
}
```

然后 smb 主题有一个默认的样式：

```less
.tiny-button + .tiny-button {
  margin-left: 8px;
}
```

这两个样式一起组合在一起，就造成了，如果两个按钮并排，并且前面一个按钮是默认按钮。
那么这两个按钮中间的间距是 16px，而不是 8px。
可以抽象一下，看看这个共性问题如何解决

File: packages/settings/events/src/components/BindEventsDialog.vue [hunk 2]
```diff
@@ -227,6 +231,20 @@ export default {
 </script>
 
 <style lang="less" scoped>
+.bind-event-dialog {
+  z-index: 99;
```
Comment (chilingling) [inline packages/settings/events/src/components/BindEventsDialog.vue:235]: 这个 `z-index` 的作用是什么？我看 tiny-vue 会自动添加 z-index
image

File: packages/settings/events/src/components/BindEventsDialogContent.vue [hunk 3]
```diff
@@ -163,14 +146,11 @@ export default {
...
-    margin-bottom: 12px;
+    margin-bottom: 8px;
     .set-params-tip {
-      margin-right: 3px;
-    }
-    .set-switch {
-      width: 60px;
-      margin-left: 10px;
+      margin-top: 6px;
+      font-weight: 400;
```
Comment (chilingling) [inline packages/settings/events/src/components/BindEventsDialogContent.vue:152]: 看看要不要用变量 `--te-base-font-weight-4`

**Review by chilingling — APPROVED** [review]

**Review by hexqi — COMMENTED** [review]
image
这块间距有点大，后面可以再调整下

**Review by hexqi — APPROVED** [review]

**Pull request merged**